src/SFA.DAS.CandidateAccount.Api/SFA.DAS.CandidateAccount.Api.http

```diff
@@ -0,0 +1,6 @@
+@SFA.DAS.CandidateAccount.Api_HostAddress = http://localhost:5177
+
+GET {{SFA.DAS.CandidateAccount.Api_HostAddress}}/weatherforecast/
```

Comment:
There is a wee weatherforecast reference here (also the test fails with a 404 error)